force-app/main/default/flows/Create_EDRD_Case.flow-meta.xml

```diff
@@ -0,0 +1,1462 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Flow xmlns="http://soap.sforce.com/2006/04/metadata" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <apiVersion>59.0</apiVersion>
+    <choices>
+        <name>AC_Review</name>
+        <choiceText>AC Review</choiceText>
+        <dataType>String</dataType>
+        <value>
+            <stringValue>AC Review</stringValue>
+        </value>
+    </choices>
+    <choices>
+        <name>Approved</name>
+        <choiceText>Approved</choiceText>
+        <dataType>String</dataType>
+        <value>
+            <stringValue>Approved</stringValue>
+        </value>
+    </choices>
+    <choices>
+        <name>Dentist</name>
```

Comment:
@Sumancdey Could you update the Create EDRD Case flow and remove the choice values we are not leveraging, for example Dentist, Midwife, etc?